src/shipit_workflow/shipit_workflow/api.py

```diff
@@ -158,6 +158,7 @@ def abandon_release(name):
         for phase in filter(lambda x: x.submitted, release.phases):
             actions = fetch_actions_json(phase.task_id)
             action_task_id, action_task, context = generate_action_task(
+                decision_task_id=phase.task_id,
```

Comment:
Which is this? the original decision task?

Reply:
This is the task-id of the task where the action was defined. Which in this case is the action task.